### PR TITLE
Txpool options

### PR DESCRIPTION
**Pull request by bgravenorst**
#568 # Pull request checklist

Document missing transaction pool CLI options.

## Impacted parts

For content changes:

- [x] Documentation content
- [ ] Documentation page organization

## After creating your PR and tests have finished

Make sure that:

- [x] You've fixed any issues raised by the tests.
- [x] You've [previewed your changes on Read the Docs](https://wiki.hyperledger.org/display/BESU/Preview+the+documentation)
  and added a [preview link](#preview).

## Preview

https://hyperledger-besu--1339.org.readthedocs.build/en/1339/public-networks/reference/cli/options/#tx-pool-disable-locals

### CODE REVIEW

File: docs/public-networks/reference/cli/options.md [hunk 1]
```diff
@@ -3358,6 +3358,66 @@ Use the [`miner_changeTargetGasLimit`](../api/index.md#miner_changetargetgaslimi
...
+    BESU_TX_POOL_DISABLE_LOCALS=true
+    ```
+
+=== "Configuration file"
+
+    ```bash
+    tx-pool-disable-locals=true
+    ```
+
+Enable or disable whether transactions sent via RPC must have the same checks, and should not be prioritized
```
Comment (macfarla) [inline docs/public-networks/reference/cli/options.md:3387]:
```suggestion
If this option is enabled, transactions sent via RPC must have the same checks, and should not be prioritized
```

Comment (macfarla) [reply docs/public-networks/reference/cli/options.md:3387]:
```suggestion
If this option is set to true, transactions sent via RPC must have the same checks, and should not be prioritized
```

**Review by macfarla — APPROVED** [review]
suggestion on wording for the disable-locals option

File: docs/public-networks/reference/cli/options.md [hunk 2]
```diff
@@ -3358,6 +3358,66 @@ Use the [`miner_changeTargetGasLimit`](../api/index.md#miner_changetargetgaslimi
...
+    BESU_TX_POOL_DISABLE_LOCALS=true
+    ```
+
+=== "Configuration file"
+
+    ```bash
+    tx-pool-disable-locals=true
+    ```
+
+If this option is set to true, transactions sent via RPC must have the same checks, and should not be prioritized
```
Comment (macfarla) [inline docs/public-networks/reference/cli/options.md:3387]:
```suggestion
If this option is set to true, transactions received via RPC must have the same checks, and should not be prioritized
```

File: docs/public-networks/reference/cli/options.md [hunk 3]
```diff
@@ -3358,6 +3358,66 @@ Use the [`miner_changeTargetGasLimit`](../api/index.md#miner_changetargetgaslimi
...
+    BESU_TX_POOL_ENABLE_SAVE_RESTORE=true
+    ```
+
+=== "Configuration file"
+
+    ```bash
+    tx-pool-enable-save-restore=true
+    ```
+
+Enable or disable whether to save the transaction pool contents to a file on shutdown and reload it at startup.
```
Comment (alexandratran) [inline docs/public-networks/reference/cli/options.md:3416]:
```suggestion
Enables or disables saving the transaction pool contents to a file on shutdown and reloading it at startup.
```

File: docs/public-networks/reference/cli/options.md [hunk 4]
```diff
@@ -3480,6 +3540,37 @@ The price bump percentage to replace an existing transaction. The default is 10.
...
+
+=== "Configuration file"
+
+    ```bash
+    tx-pool-save-file="/home/me/me_node/node_txpool.dump"
+    ```
+
+Path to the file that stores the transaction pool's content if the save and restore functionality is enabled
+using [`--tx-pool-enable-save-restore`](#tx-pool-enable-save-restore). The
+file gets created on shutdown and reloaded during startup. The default file name is `txpool.dump` in the
```
Comment (alexandratran) [inline docs/public-networks/reference/cli/options.md:3571]:
```suggestion
file is created on shutdown and reloaded during startup. The default file name is `txpool.dump` in the
```

**Review by alexandratran — APPROVED** [review]
Minor suggestions.

**Pull request merged**